README.md

```diff
@@ -63,7 +63,10 @@ Once you've created your app, you should see your Client ID and Client Secret on
 
 Now that we have them, we can try that request again with our app credentials. Enter the URL with your client id and secret into your browser:
 
-`https://api.foursquare.com/v2/venues/40a55d80f964a52020f31ee3?client_id=YOUR_CLIENT_ID&client_secret=YOUR_SECRET&v=20150201`
+`https://api.foursquare.com/v2/venues/40a55d80f964a52020f31ee3?client_id=YOUR_CLIENT_ID&client_secret=YOUR_SECRET&v=20150201&v=20140806
+  &m=foursquare
```

Comment:
Leaving off the `m=foursquare` parameter doesn't result in an error for me, but I've included it anyway in https://github.com/learn-co-curriculum/apis-and-postman/pull/5 along w/ a note